ui/ducks/send/send.js

```diff
@@ -2011,6 +2041,53 @@ export function updateSendAsset(
           } with symbol ${state.metamask.provider?.ticker ?? ETH}`,
         ),
       );
+      if (state.UNS) {
```

Comment:
Can you please describe what this section of code is doing? From my perspective it appears as though it is changing the asset based on the domain name resolved by UNS, is that accurate? It also appears to be changing the recipient?

Reply:
when you swap a token after selecting an address we need to pull the correct crypto address associated with the Unstoppable Domain. Because you can have multiple crypto addresses associated with one of our Domains we need to pull the correct one depending on which token you swap to